tachyon/zk/plonk/halo2/prover_test.h

```diff
@@ -6,6 +6,7 @@
 
 #include "gtest/gtest.h"
 
+#include "tachyon/math/elliptic_curves/bn/bn254/bn254.h"
 #include "tachyon/math/elliptic_curves/bn/bn254/g1.h"
 #include "tachyon/math/elliptic_curves/bn/bn254/g2.h"
```

Comment:
Do we still need this?

Reply:
You're right, we don't need them. I'll remove them.